server/src/testFixtures/java/io/crate/testing/TestingHelpers.java

```diff
@@ -426,7 +426,9 @@ public static Map<String, Object> toMapping(BoundCreateTable boundCreateTable) {
             boundCreateTable.analyzedTableElements().indicesMap(),
             boundCreateTable.partitionedBy(),
             tableColumnPolicy,
-            boundCreateTable.routingColumn()
+            boundCreateTable.routingColumn(),
+            -1,
+            null
```

Comment:
I can actually pass cluster state and oid (not sure but maybe can make it reset to 0 per unit-test) and it will enable a lot of extra tests.

Since this is not production code and touches a lot of files, will be done as a follow up to keep the scope of this PR smaller. If making testing code assign OID spots she bugs I will update feature branch accordingly.